services/document-manager/backend/app/config.py

```diff
@@ -15,7 +15,7 @@ class Config(object):
     ENVIRONMENT_NAME = os.environ.get('ENVIRONMENT_NAME', 'dev')
 
     SQLALCHEMY_DATABASE_URI = DB_URL
-    SQLALCHEMY_TRACK_MODIFICATIONS = True
+    SQLALCHEMY_TRACK_MODIFICATIONS = False
```

Comment:
What does this do?

Reply:
Flask-SQLAlchemy has its own event notification system that gets layered on top of SQLAlchemy. It's buggy, takes extra resources, and most people don't use it. We don't use it. It will default to being disabled in the future.